src/test/fixtures/oscap.ts

```diff
@@ -4,13 +4,16 @@ import {
 } from '../../store/imageBuilderApi';
 
 export const distributionOscapProfiles = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We can probably just remove the unused variables here instead of commenting out the warning. Would that work?